### ISSUE TITLE
SealTaskScheduler raising error in the 6.5 build

### ISSUE THREAD

**Issue by kerekesp** [ariacom/Seal-Report#62]
Hi
 the version 6.5 SealTaskScheduler.exeraising the following error:
Error:
  An assembly specified in the application dependencies manifest (SealTaskScheduler.deps.json) was not found:
    package: 'Microsoft.Win32.SystemEvents', version: '5.0.0'
    path: 'runtimes/win/lib/netcoreapp3.0/Microsoft.Win32.SystemEvents.dll'
(with 6.1 there is no such problem at me)
Can you create a new build resolving this issue?
Thank you
Regards,
Peter

**Comment by ariacom** [comment]
Thanks, I just removed the Nuget package  PackageReference Include="Microsoft.Windows.Compatibility" Version="5.0.2"  from the TaskScheduler and WindowsServoice project, this seems to update correctly the desp.json file.
By new build, you want a new setup executable ?

**Comment by kerekesp** [comment]
Hi, thanks. 
A new SealTaskScheduler.exe is enough but the new setup are making the life easier for the others installing later the 6.5.

**Comment by ariacom** [comment]
However, we had no problem to execute the Task Scheduler from fresh Windows installation...
On which configuration did you have the error ?
Did you install Net 5.0 (Microsoft Windows Desktop Runtime 5) first ?

**Comment by kerekesp** [comment]
Hi, I've tried both order, on a fresh VM,  the result is the same I have 5.0.14.30911 .NET (this can be downloaded actually)
(I tried with x86 however the seal setup supports only x64)
As I said the 6.1 version with the same .NET running without problems.

**Comment by ariacom** [comment]
I have updated to 6.5.1 which should fix the problem. Thank you to check if the new setup if fine in your environments.

**Comment by kerekesp** [comment]
Hi, 
Thanks, tested, working as expected,

Only one comment related to scheduled output. I forgot to save the reports after the new output and schedule has been created.
Then I tried to test using Task scheduler (run) A blank window comes up, closes, then after a refresh in Task Scheduler the job dissapeared. Then repeating everything as before but when the Scheduler task has been created the I tried to run it from cmd, And I got this:
C:\Program Files\Seal Report>SealTaskScheduler.exe 557cfdf-0ee8-4a8b-bf56-e05856f7ff1c
Error got when executing schedule '557cfdf-0ee8-4a8b-bf56-e05856f7ff1c':
Unable to find schedule '557cfdf-0ee8-4a8b-bf56-e05856f7ff1c' in report 'C:\ProgramData\Seal Report Repository\Reports\NAV reports.srex'.
Schedule has been deleted

   at Seal.Model.ReportExecution.InitReportSchedule(String scheduleGUID, Report& report, ReportSchedule& schedule)
   at Seal.Model.ReportExecution.ExecuteReportSchedule(String scheduleGUID, Report refReport, ReportSchedule refSchedule)

Finally I've realized that because the output, schedule was not saved in report designer, then when the task execution is failing then it removes the schedules from Windows Task scheduler. (but later when I save the report the schedule remains there without Windows task). 
So some notifikation during the schedule/output creation/modification would be usefull to remind to save the project before the first schedule execution.
